projects/widgets/docs/introduction.md

```diff
@@ -1,9 +1,55 @@
-<!-- focus: false -->
+# Widgets
 
-![](../assets/images/logo.png)
+A warm welcome to our Widgets documentation! 👋
 
-## Widgets
+Widgets are ready-made HTML/JS snippets that you can include in your own website to quickly provide a search page for cultural and leisure activities.
 
-A warm welcome to our Widgets documentation! 👋
+publiq provides a widget builder / wizard where you can easily configure your widget. without the need for advanced technical knowledge. During the configuration process all settings are saved on a scripttag. When embedded in a website or app, the scripttag will render the configured agenda with cultural activities.
+
+Widgets can be highly customized and finetuned to your own needs:
+
+* with a query you can define very precisely which events are published in your agenda
+* by simply checking and unchecking checkboxes you can determine which event information you want to show and which you don't
+* there is a broad range of styling options, through CSS or pre-defined themes
+* ...
+
+## Who are widgets (not) for?
+
+### Who are widgets for
+
+Widgets are a very accessible and versatile tool that can appeal to a wide range of partners. Widgets are of particular interest to partners that
+
+* [x] are looking for an easy but robust solution
+* [x] don't have much time or budget for the configuration or development of an agenda
+* [x] don't have the technical knowledge to develop an agenda themselves
+* [x] want a solution that is constantly kept up to date with new developments within the publiq ecosystem. New fields or features are automatically activated within a widget agenda
+
+> publiq can assist with the configuration of widgets or guide you through certain aspects (scope, styling, etc.) so that you can quickly set up an agenda tailored to the target group of your organization.
+>
+> [Contact us](mailto:jeroen@publiq.be)
+
+### Who are widgets not for
+
+In some use cases it might be appropriate to integrate with Search API instead of Widgets. We advice to integrate with our Search API instead of widgets if:
```

Comment:
```suggestion
In some use cases it might be appropriate to integrate with UiTdatabank's [Search API](https://docs.publiq.be/docs/uitdatabank/87dec20235b44-search-api) instead of Widgets. We advice to integrate with our Search API instead of widgets if:
```

Just for those who don't know what Search API is or how to find more info about it